### PR TITLE
Added init-status back, fixes #130

### LINKED ISSUE #130: 0.14.3 install bug w/ init-status
Anyone else getting this?

```sh
❯ npm i bundlesize -D
npm ERR! path /Users/scoope7/ngx-trend/node_modules/bundlesize/src/init-status.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/scoope7/ngx-trend/node_modules/bundlesize/src/init-status.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/scoope7/.npm/_logs/2017-08-31T17_22_26_445Z-debug.log
```
**Please mention other relevant information.**
- node version  
v8.3.0
- npm version  
5.4.0
- Operating system  
osx
- bundlesize version  
0.14.3
- CI you are using  
travis, but this is on my local

already tried verifying npm cache and then i cleared it just because. Using 0.14.2 for now.

### PR DESCRIPTION
**Pull request by palashmon**
## Description
Added missing [init-status](https://github.com/siddharthkp/bundlesize/blob/v0.14.2/src/init-status.js) file in [v0.14.3](https://github.com/siddharthkp/bundlesize/compare/v0.14.2...v0.14.3)

## Motivation and Context
Closes issue #130

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] If my change requires a change to the documentation I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I created an issue for the Pull Request

### CODE REVIEW

**Comment by palashmon** [comment]
@SaraVieira Please review the changes once and release a new version for testing.

```sh
npm i bundlesize@0.14.2
```
works fine, but

```sh
npm i bundlesize
```
was giving this error.

File: src/init-status.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-unused-vars */
+const build = require('./build')
```
Comment (ForsakenHarmony) [inline src/init-status.js]: Just do a plain require without a variable

Comment (palashmon) [reply src/init-status.js]: Done.

**Comment by ForsakenHarmony** [comment]
Just gonna merge this now, because removing it was probably an accident (I thought it was useless at first as well)

**Comment by palashmon** [comment]
We still need to create a new release v0.14.4, otherwise we will still get `npm install` error.

**Comment by reznord** [comment]
@SaraVieira will do it 😄 only she has the rights to publish ;)

**Comment by palashmon** [comment]
Ok. Wasn't aware of that. Anyways, I wanted to know, is there way to test if the `npm install` is working fine or not after a new release automatically?

**Comment by SaraVieira** [comment]
@reznord @palashmon Published in 0.14.4 🎉

**Comment by SaraVieira** [comment]
https://github.com/siddharthkp/bundlesize/releases/tag/v0.14.4

**Comment by palashmon** [comment]
Tested with latest release v0.14.4
```sh
npm i bundlesize -D
```
It is working fine now. No errors now, as it was occurring previously in #130

**Pull request merged**